MantidQt/CustomInterfaces/test/MuonAnalysisFitFunctionPresenterTest.h

```diff
@@ -223,7 +224,17 @@ class MuonAnalysisFitFunctionPresenterTest : public CxxTest::TestSuite {
     EXPECT_CALL(*m_fitBrowser, setMultiFittingMode(false)).Times(1);
     m_presenter->setMultiFitState(MultiFitState::Disabled);
   }
+  void test_setTFAsymmMode_On() {
```

Comment:
Consider adding a test that checks more about the interaction with the TF Asymm State checkbox, i.e. is it checked off when it should be (if enable multiple fit is clicked, if the `Grouping Options` tab (I am not sure if it was the grouping tab)  settings are changed)